server/fishtest/rundb.py

```diff
@@ -1254,12 +1253,10 @@ def sync_update_task(self, worker_info, run_id, task_id, stats, spsa):
 
         error = ""
 
-        def count_games(d):
-            return d["wins"] + d["losses"] + d["draws"]
-
         num_games = count_games(stats)
         old_num_games = count_games(task["stats"]) if "stats" in task else 0
         spsa_games = count_games(spsa) if "spsa" in run["args"] else 0
+        spsa_iter = (num_games - spsa_games) // 2
```

Comment:
Ruff is highlighting that `spsa_iter` is never used.

Reply:
![image](https://github.com/user-attachments/assets/e9bc5fdd-731f-4ae6-8eb2-e311372384a4)

I don't know if the widget is visible only upon a condition.

Reply:
I forgot to delete this variable. I have fixed it.